tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -2881,7 +2913,6 @@ TEST (test_tensor_filter, reopen_tflite_01_p)
   gsize in_size, out_size;
   GstTensorConfig config;
   gchar *str_launch_line, *prop_string;
-
```

Comment:
As I know, including any kind of style change is not recommended.

Reply:
I resolved the same problem in this file with the suggestion to delete the meaningless line from this PR. Should I upload it as a new PR?

Reply:
IMO, How about dividing commits in this pr? one for the style change, the other for the main purpose.

Reply:
Thank you! Updated.